Assets/Scripts/CharacterBehaviour.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CharacterBehaviour : MonoBehaviour
+{
+    //MOUSE CONTROL
+    public float lookSpeed, lookXLimit;
+    float rotationX = 0;
+
+    //MOVEMENT PROPERTIES
+    CharacterController characterController;
+    public float walkingSpeed = 5;
+    public float runningSpeed, jumpForce;
+    public bool canMove = true;
+    Vector3 moveDirection = Vector3.zero;
+
+    //PRIVATE ESSENTIALS
+    Camera playerCamera;
```

Comment:
comment doesn't reflect reality. this isnt private, but is labelled as such

Reply:
if there is no mention of public or private, Unity C# automatically defaults it as private.

Reply:
brilliant thank you